modules/localgov_services_page/config/install/field.field.node.localgov_services_page.localgov_services_parent.yml

```diff
@@ -6,15 +6,15 @@ dependencies:
     - node.type.localgov_services_landing
     - node.type.localgov_services_page
     - node.type.localgov_services_sublanding
-  module:
-    - localgov_services_navigation
+_core:
+  default_config_hash: 8pYEb1Mvn2qzDXMlwm1II8CRTlryGkwUMoo90KHzbas
```

Comment:
Don't want this I think. But probably do want the dependency on localgov_services_navigation?